### PR TITLE
docs(datepicker): add demo for  bsValueChange output

### PR DESCRIPTION
**Pull request by ghost**
add demo for  bsValueChange output related to #3614

### CODE REVIEW

File: demo/src/app/components/+datepicker/demos/changed-value/changed-value.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+})
+export class DemoDatepickerChangedValueComponent {
+  changedValue = false;
+  data: string;
+
+  constructor(private datePipe: DatePipe) {}
+
+  onValueChange(value: Event): void {
+    this.changedValue = true;
+    this.data = this.datePipe.transform(value);
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/changed-value/changed-value.ts]: Is it necessary to use DatePipe llike that? Check this https://angular.io/guide/pipes#using-pipes

File: demo/src/app/docs/demo-section-components/demo-examples-section/examples.component.ts [hunk 2]
```diff
@@ -5,8 +5,8 @@ import { ComponentExample } from '../../models/components-examples.model';
 
 @Component({
   selector: 'examples',
-  templateUrl: './examples.component.html',
-  changeDetection: ChangeDetectionStrategy.OnPush
```
Comment (IlyaSurmay) [inline demo/src/app/docs/demo-section-components/demo-examples-section/examples.component.ts]: Why did you remove this?

File: demo/src/app/components/+datepicker/datepicker-section.list.ts [hunk 3]
```diff
@@ -131,6 +132,14 @@ export const demoComponentContent: ContentSection[] = [
         component: require('!!raw-loader?lang=typescript!./demos/reactive-forms/reactive-forms.component.ts'),
         html: require('!!raw-loader?lang=markup!./demos/reactive-forms/reactive-forms.component.html'),
         outlet: DemoDatepickerReactiveFormsComponent
+      },
+      {
+        title: 'Changed value',
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/datepicker-section.list.ts]: change demo title and file names to `value change event`

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

File: demo/src/app/components/+datepicker/datepicker-section.list.ts [hunk 4]
```diff
@@ -131,6 +132,14 @@ export const demoComponentContent: ContentSection[] = [
         component: require('!!raw-loader?lang=typescript!./demos/reactive-forms/reactive-forms.component.ts'),
         html: require('!!raw-loader?lang=markup!./demos/reactive-forms/reactive-forms.component.html'),
         outlet: DemoDatepickerReactiveFormsComponent
+      },
+      {
+        title: 'Value change event',
+        anchor: 'value-change-event',
+        component: require('!!raw-loader?lang=typescript!./demos/value-change-event/value-change-event.ts'),
+        html: require('!!raw-loader?lang=markup!./demos/value-change-event/value-change-event.html'),
+        description: `<p>You can subscribe to datepicker's changing events</p>`,
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/datepicker-section.list.ts]: You can subscribe to datepicker's value change event

File: demo/src/app/components/+datepicker/demos/value-change-event/value-change-event.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-datepicker-value-change-event',
+  templateUrl: './value-change-event.html'
+})
+export class DemoDatepickerValueChangeEventComponent {
+  changedValue = false;
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/value-change-event/value-change-event.ts]: no need for this

File: demo/src/app/components/+datepicker/demos/value-change-event/value-change-event.html [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<div class="row">
+  <div class="col-xs-12 col-12 col-sm-6 col-md-4 form-group">
+    <h4 *ngIf="changedValue">Changed date is {{data | date}}</h4>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/value-change-event/value-change-event.html]: *ngIf="data"

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — APPROVED** [review]

File: demo/src/app/components/+datepicker/demos/value-change-event/value-change-event.ts [hunk 7]
```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-datepicker-value-change-event',
+  templateUrl: './value-change-event.html'
+})
+export class DemoDatepickerValueChangeEventComponent {
+  data: object;
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/value-change-event/value-change-event.ts]: change type to Date

File: demo/src/app/components/+datepicker/demos/value-change-event/value-change-event.ts [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+  onValueChange(value: Event): void {
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/value-change-event/value-change-event.ts]: same

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Make output more looks like output, not like another one title. 
outputtitle

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good
